system_tests/test_system.py

```
@@ -131,52 +166,156 @@ def test_database_handling(self):
             }
         ]
 
+    def test_loads_default_tables(self):
+        """Test loading default tables."""
         sleep(5.0)  # wait until default tables are loaded
```

Mabey we can check with a busy-wait when the tables will be loaded and print it (We could also ask the hyrise directly for that and then check the endpoint). We could still let the test fail if it would take more than 5 seconds.

---

I think, because of using busy-wait, we will not get precise time values anyway. So, what would be the use case of printing them?

---

It would be just a pinpoint.